src/components/Button/Button.scss

```diff
@@ -277,16 +308,29 @@ $partial-button-filled-pressed-box-shadow: inset 0 0 0 0 transparent,
     @include high-contrast-button-outline;
   }
 
+  &:focus:not(:active) > .Content {
```

Comment:
Side note there are a few places where I did `&:focus { @include focus-ring }` and `&:active { @include no-focus-ring }`. Those can likely be refactored to `&:focus:not(:active) { @include focus-ring }`.